its/src/test/java/com/sonar/maven/it/suite/MavenTest.java

```diff
@@ -442,4 +444,27 @@ void supportMavenEncryption() throws Exception {
     ORCHESTRATOR.executeBuild(build);
   }
 
+  @Test
+  void supportMavenEncryptionWithDefaultSecuritySettings() throws Exception {
+
+    wsClient.settings().set(new SetRequest().setKey("sonar.forceAuthentication").setValue("true"));
+    wsClient.users().create(new CreateRequest().setLogin("julien2").setName("Julien2").setPassword("123abc"));
+
+    MavenBuild build = MavenBuild.create(ItUtils.locateProjectPom("maven/maven-only-test-dir"))
+            .setGoals(cleanSonarGoal());
+
+    File securityXml = new File(this.getClass().getResource("/security-settings.xml").toURI());
+    File settingsXml = new File(this.getClass().getResource("/settings-with-encrypted-sonar-password.xml").toURI());
+
+    String userHomeDir = System.getProperty("user.home");
+    Path defaultPath = Paths.get(userHomeDir, ".m2", "settings-security.xml");
+    Files.copy(securityXml.toPath(), defaultPath);
+
+    build.addArgument("--settings=" + settingsXml.getAbsolutePath());
+
+    build.setProperty("sonar.login", "julien");
+    build.addArgument("-Psonar-password");
```

Comment:
Not a big deal: it would be nice to use the long option here
```suggestion
    build.addArgument("--activate-profiles sonar-password");
```

Reply:
yep, that's copy/paste from the test above, I'll update in both places

Reply:
Actually I can't add it as an argument as it will contain space, let's keep it as is for now